tests/Config/ConfigTest.php

```
@@ -118,25 +128,25 @@ public function testGetConfig(
+        if (str_starts_with($key, 'global-')) {
+            $this->io->expects(self::atLeast(2))->method('isDebug')->willReturn(true);
 
             $globalLogComposer = false;
             $globalLogConfig = false;
 
             $this->io
-                ->expects($this->atLeastOnce())
+                ->expects(self::atLeastOnce())
                 ->method('writeError')
                 ->willReturnCallback(
                     static function ($message) use ($globalPath, &$globalLogComposer, &$globalLogConfig): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$message'. (undefined)

(UnusedFormalParameter)